raytrace.py

```diff
@@ -185,13 +193,14 @@ def change_of_direction(self, ray, normal_vector):
 
 
 class VolumeMaterial(Material, object):
-    def __init__(self, name, properties):
+    def __init__(self, name, index_of_refraction, extinction_coefficient = None):
```

Comment:
I would keep the old signature, more generic than the new one. This way, if you need to introduce new parameters, you don't have to redefine the __init__ method.